Platform/Qemu/SbsaQemu/SbsaQemu.dsc

```diff
@@ -392,10 +396,18 @@ DEFINE NETWORK_HTTP_BOOT_ENABLE       = FALSE
   gArmTokenSpaceGuid.PcdVFPEnabled|1
 
   # System Memory Base -- fixed
-  gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10000000000
+  !if $(ENABLE_RME)
+    #
+    # When RME is enabled the RMM is installed at 0x10000000000
+    # and the system base memory bumped by 1072MB.
+    #
+    gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10043000000
+  !else
+    gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10000000000
+  !endif
 
   # Space for 32 stacks
-  gArmPlatformTokenSpaceGuid.PcdCPUCoresStackBase|0x1000007c000
+  gArmPlatformTokenSpaceGuid.PcdCPUCoresStackBase|gArmTokenSpaceGuid.PcdSystemMemoryBase+7c000

```

Comment:
It should be "0x7c000" with spaces on both sides of "+" sign.

Reply:
Yes of course - thanks for pointing that out.